client/blocks/reader-full-post/_content.scss

```diff
@@ -795,3 +795,8 @@
     padding-left: 16px;
     padding-right: 16px;
 }
+
+// Import styles for Jetpack Story block.
+// This is a copy of Jetpack's extensions/blocks/story/player/style.scss, with
+// some unnecessary styles removed.
+@import 'story.scss';
```

Comment:
I think it's best to keep the imports at the top of the file, for readability.

Reply:
I was wondering about that - since it just inlines the css in the output I wasn't sure about putting stories at the top, but I'm happy to do it whichever way. Updated in a0f4fbc.